### DISCUSSION TITLE
开启mako后，chainWebpack下插件会失效是吗

### DISCUSSION THREAD

**Discussion by wangyongqibj** [umijs/umi#12724] [Q&A]
为了编译时eslint检查，使用了eslint-webpack-plugin，使用mako后失效

**Comment by stormslowly** [comment]
是的，建议使用 git hook or  CI 的方式进行 eslint 检查的卡点，这样就和 bundler 无关了

**Comment by ncast-dev** [comment]
## DIGI-374 — PO worker pool: décompositions d'idées en parallèle

**Status:** DONE ✅
**Commit:** 6b994af

### Ce qui a été livré

- `shared/decompose_jobs.py` — queue SQLite pour les jobs de décomposition (calqué sur `review_jobs.py`) : `enqueue_job`, `take_next_for_worker`, `mark_done`, `list_pending`, `register_worker`, `heartbeat_worker`
- `po_worker/main.py` — daemon worker avec `decompose_one_job` + `run_daemon` (même pattern que `reviewer_worker`)
- `po_worker/__init__.py` + `po_worker/__main__.py` — allow `python -m po_worker`
- `tests/test_DIGI-374.py` — 12 tests, tous verts

### Tests
```
12 passed in 0.72s
```

Régressions DIGI-405 : 11 passed ✅
